changelogs/3.0.1.md

```diff
@@ -0,0 +1,48 @@
+# 3.0.1
```

Comment:
We should strip changelog entries that reside in the separate files, when compose them into release notes. See https://github.com/tarantool/tarantool/blob/master/doc/changelogs.md#generate-changelog-file

Reply:
Almost broken with "strip" in relation to files removal :)
Fixed.